src/open_inwoner/scss/components/Form/Submit.scss

```diff
@@ -10,7 +11,7 @@
     background-color 300ms;
 
   &:hover {
-    background-color: darken(#248641, 5);
+    background-color: color.adjust(#248641, $lightness: -5%);
```

Comment:
I don't think the .submit class is actually used anywhere (?). We do have Type=submit for the buttons, but they seem to all use `--color-primary-darker` as hover background. But i guess it's interesting to see the modern syntax in action.